tests/Type/ContentItemRegionTypeTest.php

```diff
@@ -1,56 +1,56 @@
 <?php
 /**
- * @author Gerard van Helden <gerard@zicht.nl>
  * @copyright Zicht Online <http://zicht.nl>
  */
 
 namespace ZichtTest\Bundle\PageBundle\Assets {
-    class foo {}
-    class bar {}
+    class foo
+    {
+    }
+    class bar
+    {
+    }
 }
 
- namespace ZichtTest\Bundle\PageBundle\Type {
-
+namespace ZichtTest\Bundle\PageBundle\Type {
      use PHPUnit\Framework\TestCase;
      use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 
     class ContentItemRegionTypeTest extends TestCase
     {
-        function testConstruct()
+        public function testConstruct()
         {
             $ret = new \Zicht\Bundle\PageBundle\Type\ContentItemRegionType(
                 'foo',
-                array('a' => 'a', 'b' => 'b'),
+                ['a' => 'a', 'b' => 'b'],
                 $this->createMock('Symfony\Component\Translation\TranslatorInterface')
             );
 
-            $this->assertEquals('zicht_content_item_region', $ret->getName());
+            $this->assertEquals('zicht_content_item_region', $ret->getBlockPrefix());
```

Comment:
This test doesn't test much functionality. Only if PHP can construct a certain class and if you can call a method. The method name depends on the Symfony interface. So nothing tested here of the actual page bundle...